ext-apiserver/pkg/controller/config/configuration.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"github.com/spf13/pflag"
+)
+
+type Configuration struct {
+	Kubeconfig   string
+	Cloud        string
+	KubeadmToken string
```

Comment:
This seems like configuration specific to our actuator, and is probably better stored there. Not every deployer will even use kubeadm.

Reply:
Agree. My current focus is to have ext-apiserver to be functional equivalent with CRD code so that new work can be done on top of the new code base. I will leave the refactoring for a future PR.